test/CMakeLists.txt

```diff
@@ -182,6 +182,16 @@ if(HAVE_LIBUDEV_H)
     add_definitions(-DHAVE_LIBUDEV_H)
 endif()
 
+set(FFmpeg_FIND_COMPONENTS AVCODEC AVFORMAT AVUTIL SWSCALE)
+include("${SDL3_SOURCE_DIR}/cmake/FindFFmpeg.cmake")
```

Comment:
I would have done `find_package(FFmpeg QUIET COMPONENTS AVCODEC AVFORMAT AVUTIL SWSCALE)` here, but perhaps this one is better as it won't conflict with another `FindFFMPEG.cmake`.